### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by nabucopena**
-Now there is an avatar on the right corner.
-We use cgi to escape html and avoid injections.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -62,6 +67,21 @@ def conn
...
   
   on !!current_user do
+
+    on "settings" do
+      on get do
+        render("settings")
+      end
+      on post do
+        on param("url") do |url|
+          result = Accounts.change_avatar(url, current_user.fetch(:id))
```
Comment (carlosipe) [inline app.rb:77]: I think I'd use keyword arguments here, something like:

`Accounts.change_avatar(user_id: current_user.fetch(:id), url: url)`

In that way you don't have to remember the order of the arguments: "Was it first url and then user_id or first user_id and then url? 🤔 "

File: app.rb [hunk 2]
```diff
@@ -62,6 +67,21 @@ def conn
...
+          session[:error] = result.fetch(:error)
+          res.redirect(result.fetch(:redir))
```
Comment (carlosipe) [inline app.rb:79]: Don't shorten the names, what does `redir` means?

On the other hand, why does account know about the web urls?

File: models.rb [hunk 3]
```diff
@@ -92,15 +95,34 @@ def self.list_positions
 end
 
 class Accounts
+
+  def self.change_avatar(url, id)
+    begin
+      response = Requests.request("GET", url)
+      if FastImage.type(response) == ":png"
+        File.write("public/#{id}.png", response.body)
+        redir = "/games"
```
Comment (carlosipe) [inline models.rb:104]: I wouldn't expect the `Accounts` class to know anything about urls. Remember that this class might be used in a context outside the web (e.g: from the terminal).

I'd expect the decision of which route to redirect to, to belong to the web side (app.rb). `Accounts` should return "agnostic" responses that are clear to enough for the web part to take its own decisions of where to redirect the user to.

File: models.rb [hunk 4]
```diff
@@ -92,15 +95,34 @@ def self.list_positions
 end
 
 class Accounts
+
+  def self.change_avatar(url, id)
+    begin
+      response = Requests.request("GET", url)
+      if FastImage.type(response) == ":png"
```
Comment (carlosipe) [inline models.rb:102]: Are we only allowing `png` files? What about jpg?

Comment (nabucopena) [reply models.rb]: i'll do it in a new PR, i just allowed png to handle easier

File: app.rb (hunk 1 above)
Comment (nabucopena) [reply app.rb]: done

File: app.rb (hunk 2 above)
Comment (nabucopena) [reply app.rb]: now it only returns the error, and the controller selects where to redirect

File: models.rb (hunk 3 above)
Comment (nabucopena) [reply models.rb]: done

File: Gemfile [hunk 5]
```diff
@@ -7,4 +7,7 @@ gem "mote"
 gem "pg"
 gem "bcrypt"
 gem "puma"
-gem "cutest"
\ No newline at end of file
+gem "cutest"
```
Comment (carlosipe) [inline Gemfile:10]: Why is cutest here in this PR?

**Pull request merged**